testbed/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```

Comment:
This file must contains feed data

Reply:
I'm loading data through url!,
isnt it right ?
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/

Reply:
No. see issue
https://github.com/webRunes/WRIO-InternetOS/issues/1610
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/ is a sensor page
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed is a feed page (sensor's data) of the sensor

Reply:
https://github.com/WRIO-Hubs/imec.wr.io/ already contains example pages.

Reply:
Sorry, I was talking about 
ttps://imec.wr.io/a575d13e-b559-4d19-bb2a-1a3cd67952e7/feed
here I am getting data through url.
or simply I should add all data in this file ?
like this = https://raw.githubusercontent.com/WRIOdev/sensorID-a575d13e-b559-4d19-bb2a-1a3cd67952e7/gh-pages/index.html

Reply:
Yes, add feed data.

Reply:
Ok

Reply:
what is the pupose of adding the url here, when I will be adding feed data directly ?